platform/linuxbsd/wayland/display_server_wayland.h

```diff
@@ -100,6 +100,8 @@ class DisplayServerWayland : public DisplayServer {
 
 		String title;
 		ObjectID instance_id;
+
+		TypedArray<Rect2i> passthrough_rectangles;
```

Comment:
The method defines work fine, but I had issues with the window `mouse_passthrough_rects` property binding when using Vector<>:
```cpp
ADD_PROPERTY(PropertyInfo(Variant::ARRAY, "mouse_passthrough_rects", PROPERTY_HINT_ARRAY_TYPE, "Rect2i"), "set_mouse_passthrough_rects", "get_mouse_passthrough_rects");
```
so I was forced to use TypedArray instead. I think the reason otherwise vectors work is because the bindings there use PackedVector2Array, but there is no equivalent for Rect2i.